Packs/CiscoSpark/.pack-ignore

```diff
@@ -1,5 +1,8 @@
 [file:CiscoSpark.yml]
-ignore=IN145
+ignore=IN145, IN140
```

Comment:
Please remove this - IN140 - you can't skip it
Again, next time check in the demisto-sdk repo